### ISSUE TITLE
Integration problem with MDL (Material Design Lite)

### ISSUE THREAD

**Issue by JoaoPedroXD** [matthieua/WOW#291]
Dear,

I would like to know how I can work with **WOW.js** along with **Material Design Lite** (**MDL**), as I have been trying for weeks and I have not made any progress without having to completely remove the layout.

Follow the **code**:

`<!DOCTYPE html>

<html>
    <head>
        <title>U@O Nome - Início</title>
        <link rel="icon" type="image" href="imagens/logo_reduzida.png"/>
        <meta charset="UTF-8">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">
        <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
        <link rel="stylesheet" href="CSS/Estilo.css" type="text/css">
        <link rel="stylesheet" href="CSS/animate.css" type="text/css">
        <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
        <script src="js/wow.js"></script>
        <link rel="stylesheet" href="https://code.getmdl.io/1.3.0/material.blue_grey-red.min.css" />
        <script defer src="https://code.getmdl.io/1.3.0/material.min.js"></script>
        <script src="classes/ClasseGeral1.js"></script>
        <script>
            function carrega(){
                document.getElementById("carregando").style = "display: none";
                document.getElementById("tudo").style = "visibility: visible";
            }
            
            function incarrega(){
                document.getElementById("carregando").style = "display: visible";
                document.getElementById("tudo").style = "display: none";
            }
        </script>
        <?php include_once("analyticstracking.php") ?>
    </head>
    <body onload="carrega()">
        <div id="carregando" class="carregando"></div>
        
        <div class="mdl-layout mdl-js-layout mdl-layout--fixed-header"
             id="tudo">
            <header class="mdl-layout__header">
                <div class="mdl-layout__header-row">
                
                <span class="mdl-layout-title">
                    <h2 id="titulo">
                        <img src="imagens/logo_reduzida.png" 
                                 width="auto" height="50px" id="logo-header"/>
                        Nome
                    </h2></span>
                </div>
                
                <div class="mdl-layout__tab-bar mdl-js-ripple-effect">
                  <a href="#scroll-tab-inicio" class="mdl-layout__tab is-active">Início</a>
                  <a href="#scroll-tab-como-funciona" class="mdl-layout__tab">Como Funciona</a>
                  <a href="#scroll-tab-precos" class="mdl-layout__tab">Preços</a>
                  <a href="#scroll-tab-sobre" class="mdl-layout__tab">Sobre</a>
                  <div class="mdl-layout-spacer" style="padding-right: 60px;"></div>
                </div>
                
            </header>
                <div class="mdl-layout__drawer">
                    <span class="mdl-layout-title">
                        <img src="imagens/logo_completa.png" id="logo-header-menu"/>
                    </span>
                    <nav class="mdl-navigation">
                        <a class="mdl-navigation__link" href="index.php" onclick="incarrega()">Início</a>
                        <a class="mdl-navigation__link" href="login.php" onclick="incarrega()">Login</a>
                        <a class="mdl-navigation__link" href="atualizacoes.php" onclick="incarrega()">Atualizações</a>
                        <a class="mdl-navigation__link" href="ajuda.php" onclick="incarrega()">Ajuda</a>
                        <a class="mdl-navigation__link" href="termos.php" onclick="incarrega()">Termos do Software</a>
                    </nav>
                </div>
            
            
            <div class="mdl-layout__content">
                <section class="mdl-layout__tab-panel is-active" id="scroll-tab-inicio">
                    <div class="page-content wow lightSpeedIn">
                        <h2>Início</h2>
                        <h4>
                            
                        </h4>
                    </div>
                    <div class="page-content wow lightSpeedIn">
                        <h2>Início</h2>
                        <h4>
                            
                        </h4>
                    </div>
                    <div class="page-content wow lightSpeedIn">
                        <h2>Início</h2>
                        <h4>
                            
                        </h4>
                    </div>
                    <div class="page-content wow lightSpeedIn">
                        <h2>Início</h2>
                        <h4>
                            
                        </h4>
                    </div>
                    <div class="page-content wow lightSpeedIn">
                        <h2>Início</h2>
                        <h4>
                            
                        </h4>
                    </div>
                    <div class="page-content wow lightSpeedIn">
                        <h2>Início</h2>
                        <h4>
                            
                        </h4>
                    </div>
                    <div class="page-content wow lightSpeedIn">
                        <h2>Início</h2>
                        <h4>
                            
                        </h4>
                    </div>
                    <div class="page-content wow lightSpeedIn">
                        <h2>Início</h2>
                        <h4>
                            
                        </h4>
                    </div>
                </section>
                <section class="mdl-layout__tab-panel" id="scroll-tab-como-funciona">
                    <div class="page-content wow lightSpeedIn">
                        <h2>Como Funciona</h2>
                    </div>
                </section>
                <section class="mdl-layout__tab-panel" id="scroll-tab-precos">
                    <div class="page-content wow lightSpeedIn">
                        <h2>Preços</h2>
                    </div>
                </section>
                <section class="mdl-layout__tab-panel" id="scroll-tab-sobre">
                    <div class="page-content wow lightSpeedIn">
                        <h2>Sobre</h2>
                    </div>
                </section>      
                <footer class="mdl-mini-footer">
                    <div class="mdl-mini-footer__right-section">
                        <div class="mdl-logo">U@O Corporation LTDA &COPY;</div>
                        <ul class="mdl-mini-footer__link-list">
                            <li class="footer"><a href="ajuda.php">Ajuda</a></li>
                            <li class="footer"><a href="termos.php">Termos do Software</a></li>
                        </ul>
                    </div>
                </footer>
            </div>
        </div>
        <script>
            new WOW().init();
        </script>
    </body>
</html>
`

**Regards**,
**João Pedro**.

**Comment by JoaoPedroXD** [comment]
I already did it, thank you!
